static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -258,6 +244,10 @@ export function EventFeatureFlagList({
     columnTwo = truncatedItems.slice(10, 20);
   }
 
+  const extraFlags = hydratedFlags.length - 20;
```

Comment:
Is `20` a magic number in a bunch of other places? Might be good to refactor into a constant.